_data/apps.yml

```diff
@@ -2183,4 +2183,13 @@
   website: "https://englishextraapp.codeplex.com"
   repository: "https://github.com/englishextra/englishextra-app"
   icon: "englishextra-app.png"
+-
+  name: "sheepChat"
+  description: "Multi-chat for streamers"
+  website: "https://sheep.chat"
+  repository: ""
```

Comment:
Rather than specifying empty values, please remove `repository` and `keywords` entirely.